x-pack/test/idp-fixture/oidc/Dockerfile

```diff
@@ -1,9 +1,13 @@
 FROM c2id/c2id-server-demo:12.18 AS c2id
 FROM openjdk:11.0.16-jre
 
+# TODO this is obviously horrible
+RUN apt-get update -qqy && apt-get install -qqy python3
+
 COPY --from=c2id /c2id-server /c2id-server
 COPY --from=c2id /etc/c2id /etc/c2id
+COPY ./oidc/setup.sh /fixture/
 
-ENV CATALINA_OPTS="-DsystemPropertiesURL=file:///etc/c2id/override.properties"
+ENV CATALINA_OPTS="-DsystemPropertiesURL=file:///config/c2id/override.properties"
```

Comment:
Mapping this to a different target folder because the default c2id docker image comes with an empty file `//etc/c2id/override.properties` which doesn't work with the polling we have in x-pack/test/idp-fixture/oidc/setup.sh